ansible_collections/arista/avd/roles/eos_designs/docs/tables/node-type-uplink-configuration.md

```diff
@@ -155,7 +155,7 @@
 
         # Override the default `uplink_type` set at the `node_type_key` level.
         # `uplink_type` must be "p2p" if `vtep` or `underlay_router` is true for the `node_type_key` definition.
```

Comment:
This comment may have to change.

Reply:
Many other places the docs mention uplink type p2p. We need to go through all of it and evaluate.